.pre-commit-config.yaml

```diff
@@ -29,6 +29,7 @@ repos:
     hooks:
       - id: mypy
         files: ^ds-caselaw-ingester/
+        language_version: "3.11"
```

Comment:
This feels like it's snuck in here and shouldn't - we target 3.12, as 3.13 is where the Saxon incompatibility crops up.

Reply:
So, that's because I wanted it to match template.yml, which is what we're running the lambda on. I'll upgrade that and local-stack.